### ISSUE TITLE
How to read UTF8 values from MySQL 5.6?

### ISSUE THREAD

**Issue by Kipriz** [sfu-db/connector-x#262]
I have an issue with reading UTF8 data from MySQL 5.6
I need to use an external quite old database in our company, there is no way to migrate it to 8.0 or something.
When I query data from MySql 8.0, it works fine. When I use MySql 5.6 (the production database), it fails with PanicException:
`pyo3_runtime.PanicException: Could not retrieve alloc::string::String from Value`
I tried to pass any options to the connection string itself, but no options for encoding is supported.

Is there any way to resolve this?

Sample code:
```
import connectorx as cx
import pyarrow.parquet as pq

sql = """
        SELECT N'10$ - Journée de la Femme' as onsale_name
"""

conn_8_0 = "mysql://username:password@localhost:3380/control_panel"
conn_5_6="mysql://username:password@localhost:3306/control_panel"

df = cx.read_sql(conn_eventdb, sql, return_type="arrow")
pq.write_table(df, './parquet/operator_types_pyarrow.parquet')
```
In the select query above you may notice the char `é`. It is the little source of all evil =)

**Comment by wangxiaoying** [comment]
Hi @Kipriz , I tried to start a mysql 5.6 docker container pulled from [here](https://hub.docker.com/_/mysql?tab=tags) (with tag 5.6) and reproduce your error. But the query could successfully run on my side: 
<img width="1912" alt="image" src="https://user-images.githubusercontent.com/5569610/161464612-ec8c910e-d3d1-4101-871f-a6b4f9e8fe50.png">
Is it possible the error is caused by other settings of the server?

**Comment by Kipriz** [comment]
@wangxiaoying I think it is connected to the default chartset and collation.
That prod server has:
| DEFAULT_CHARACTER_SET_NAME | DEFAULT_COLLATION_NAME |
| ---- | ---- |
|latin1|latin1_swedish_ci|

Whereas usually charset and collations are the following:
| DEFAULT_CHARACTER_SET_NAME | DEFAULT_COLLATION_NAME |
| ---- | ---- |
|utf8mb4|utf8mb4_unicode_ci|

The query is `SELECT @@character_set_database, @@collation_database;`

Do you have any hints how to deal with this?

**Comment by wangxiaoying** [comment]
@Kipriz , yeah the character set and collation looks like the reason for the issue. I don't really have any experience in this but it looks like you can change them through [`alter` command](https://stackoverflow.com/questions/6115612/how-to-convert-an-entire-mysql-database-characterset-and-collation-to-utf-8). But it would be better to backup your database first if you want to do it.
